src/JustSaying/Messaging/MessageSerialization/SystemTextJsonSerializer`1.cs

```diff
@@ -0,0 +1,50 @@
+using System.Text.Json;
+using JustSaying.Models;
+
+namespace JustSaying.Messaging.MessageSerialization;
+
+/// <summary>
+/// Provides serialization and deserialization functionality for messages of type <typeparamref name="T"/> using System.Text.Json.
+/// </summary>
+/// <typeparam name="T">The type of message to be serialized or deserialized. Must inherit from <see cref="Message"/>.</typeparam>
+public sealed class SystemTextJsonMessageBodySerializer<T> : SystemTextJsonMessageBodySerializerBase, IMessageBodySerializer where T: Message
+{
+    private readonly JsonSerializerOptions _options;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="SystemTextJsonMessageBodySerializer{T}"/> class with default JSON serializer options.
+    /// </summary>
+    public SystemTextJsonMessageBodySerializer()
+    {
+        _options = DefaultJsonSerializerOptions;
+    }
```

Comment:
I'd be tempted to say we should drop this constructor in favour of exposing a default options that can be passed to the other constructor. In AoT this one will have to be an instant throw.